src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -70,6 +71,14 @@ class SegmentedOolWriter : public ExtentOolWriter {
     });
   }
 
+  paddr_t alloc_paddr(extent_len_t length) final {
+    if (segment_allocator.get_gen() > 0) {
+      return make_delayed_temp_paddr(0);
+    } else {
+      return make_record_relative_paddr(0);
```

Comment:
`ExtentOolWriter` is about out-of-line extents, inline extents should not call `ExtentOolWriter::alloc_paddr(extent_len_t)`.